src/benchq/compilation/julia_utils.py

```diff
@@ -23,6 +23,41 @@ def get_algorithmic_graph_from_graph_sim_mini(circuit):
     return graph
 
 
+def get_algorithmic_graph_from_ruby_slippers_with_hyperparams(
+    verbose=True,
+    max_graph_size=9999999,
+    teleportation_threshold=40,
+    min_neighbors=6,
+    oz_to_kansas_distance=4,
+    max_num_neighbors_to_search=99999,
+    decomposition_strategy=0,
```

Comment:
Would it make sense to replace `9999999` with `1e7` and `99999` with `1e5`? I think that would make it a bit more readable. Same goes for default values for `run_ruby_slippers`.